web/src/add_subscribers_pill.ts

```diff
@@ -132,6 +132,10 @@ export function create_without_add_button({
     return pill_widget;
 }
 
+export function append_pill_from_text(text: string, pill_widget: CombinedPillContainer): void {
```

Comment:
Can we use `user_group_pill.append_user_group` here?

Reply:
Fixed! Added a function to add a user group pill from group name for now since that is all we need (not stream or user)